custom_components/tahoma/climate_devices/atlantic_heat_recovery_ventilation.py

```diff
@@ -0,0 +1,174 @@
+"""Support for AtlanticHeatRecoveryVentilation."""
+from __future__ import annotations
+
+from typing import cast
+
+from pyoverkiz.enums import OverkizCommand, OverkizCommandParam, OverkizState
+
+from homeassistant.components.climate import ClimateEntity
+from homeassistant.components.climate.const import (
+    FAN_AUTO,
+    HVAC_MODE_FAN_ONLY,
+    SUPPORT_FAN_MODE,
+    SUPPORT_PRESET_MODE,
+)
+from homeassistant.const import TEMP_CELSIUS
+
+from ..coordinator import OverkizDataUpdateCoordinator
+from ..entity import OverkizEntity
+
+FAN_BOOST = "home_boost"
+FAN_KITCHEN = "kitchen_boost"
+FAN_AWAY = "away"
+FAN_BYPASS = "bypass_boost"
+
+PRESET_AUTO = "auto"
+PRESET_PROG = "prog"
+PRESET_MANUAL = "manual"
+
+OVERKIZ_TO_FAN_MODES: dict[str, str] = {
+    OverkizCommandParam.AUTO: FAN_AUTO,
+    OverkizCommandParam.AWAY: FAN_AWAY,
+    OverkizCommandParam.BOOST: FAN_BOOST,
+    OverkizCommandParam.HIGH: FAN_KITCHEN,
+    "": FAN_BYPASS,
+}
+
+FAN_MODES_TO_OVERKIZ = {v: k for k, v in OVERKIZ_TO_FAN_MODES.items()}
+
+
+class AtlanticHeatRecoveryVentilation(OverkizEntity, ClimateEntity):
+    """Representation of a AtlanticHeatRecoveryVentilation device."""
+
+    _attr_fan_modes = [*FAN_MODES_TO_OVERKIZ]
+    _attr_hvac_modes = [HVAC_MODE_FAN_ONLY]
+    _attr_preset_modes = [PRESET_AUTO, PRESET_PROG, PRESET_MANUAL]
+    _attr_temperature_unit = TEMP_CELSIUS
+    _attr_supported_features = SUPPORT_PRESET_MODE | SUPPORT_FAN_MODE
```

Comment:
We can use ClimateEntityFeatures here

Reply:
I will do that for core, but otherwise I need to bump the custom component version etc. :D